validator/src/main.rs

```diff
@@ -1529,8 +1529,15 @@ pub fn main() {
         packager_thread_niceness_adj: snapshot_packager_niceness_adj,
     };
 
+    // The accounts hash interval shall match the snapshot interval, unless manually specified
     validator_config.accounts_hash_interval_slots =
-        value_t_or_exit!(matches, "accounts-hash-interval-slots", u64);
+        value_t!(matches, "accounts_hash_interval_slots", Slot).unwrap_or_else(|_| {
+            std::cmp::min(
+                full_snapshot_archive_interval_slots,
+                incremental_snapshot_archive_interval_slots,
+            )
+        });
+
```

Comment:
```suggestion
        std::cmp::min(
            full_snapshot_archive_interval_slots,
            incremental_snapshot_archive_interval_slots,
        );

```

for the most part, "deprecation" has really meant "remove without breaking existing configs" wrt cli args. to that end, i don't think we should even consider any passed arg value and instead just take the `.unwrap_or_else()` value here